modules/terrain/src/terrain-loader.js

```diff
@@ -0,0 +1,29 @@
+/* global __VERSION__ */ // __VERSION__ is injected by babel-plugin-version-inline
+import loadTerrain from './lib/parse-terrain';
+
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+export const TerrainWorkerLoader = {
+  id: 'terrain',
+  name: 'Terrain',
+  version: VERSION,
+  extensions: ['png', 'pngraw'],
```

Comment:
What happens if the user registers both this loader and the image loader? We don't really have a way of disambiguating loaders at this point. 

One idea was to let the app specify what category of data it wants back (e.g. in this case 'mesh')

Reply:
Originally we were not going to register extensions for this reason, but I then added something since `validatePointCloudCategoryData`. My approach isn't a solution though, and I like the image processing idea.

Reply:
I am opposed to arbitrarily expanding the `ImageLoader`. deck.gl depends on it and this will bring significantly more code and dependencies into the package.